avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -65,6 +65,13 @@ referenceSimName =
 unit =
 
 #---------------------------------------
+## Uncomment this section FILTER in your local copy of the ini file and add filter parameter and parameter values
+## see the example provided below for release thickness
+#[FILTER]
+## define parameter and corresponding values from the simulation configuration to filter simulations
+## multiple parameters are possible, just add them in a new line each
+##relTh = 0.75|0.9|0.8
```

Comment:
if I uncomment this, it will filter me all simulations with relTH= to 0.75 or 0.9 or 0.8?

Reply:
yes